packages/components/src/utils/property-checkers/check-exists.ts

```diff
@@ -0,0 +1,7 @@
+export function checkExists<T = unknown>(value: T, error: string) {
+  const valueIsUndefined = typeof value === 'undefined';
```

Comment:
Why is this a string?

```suggestion
  const valueIsUndefined = typeof value === undefined;
```

Reply:
Because `typeof` returns a string (I used `typeof` as a [best practice](https://thisthat.dev/variable-undefined-vs-typeof-variable-undefined/))